### ISSUE TITLE
with only a TIKV cluster ?

### ISSUE THREAD

**Issue by raphaelauv** [pingcap/tidb-dashboard#1708]
## Question

Hi can I use tidb-dashboard with a TIKV only deployment ? thanks

**Comment by baurine** [comment]
> ## Question
> Hi can I use tidb-dashboard with a TIKV only deployment ? thanks

we don't support this usage yet.

**Comment by raphaelauv** [comment]
yes if I start a TidbCluster with a zero replicas to TiDB it fail

![image](https://github.com/user-attachments/assets/18f890ec-f56c-4a8c-9973-64f4d2fe9dba)
